pkg/dao/repository_config.go

```diff
@@ -306,6 +308,30 @@ func ModelToApiFields(repoConfig models.RepositoryConfiguration, apiRepo *api.Re
 	apiRepo.URL = repoConfig.Repository.URL
 }
 
+func convertOrderByToSQL(orderBy string) string {
+	order := "name asc" // default order
+
+	switch orderBy {
+	case "name asc":
+		order = "name asc"
+	case "name desc":
+		order = "name desc"
+	case "url asc":
+		order = "url asc, name"
+	case "url desc":
+		order = "url desc, name"
+	case "arch asc":
+		order = "arch asc, name"
+	case "arch desc":
+		order = "arch desc, name"
+	case "versions asc":
+		order = "cardinality(versions) asc, name"
+	case "versions desc":
+		order = "cardinality(versions) desc, name"
```

Comment:
For versions, this effectively orders by length.

Reply:
I like it, sold! 
I mean, buy me coffee and I'll do it... 

I'm bad at bargaining. 😆